package.json

```diff
@@ -15,14 +15,13 @@
     "eslint-quiet": "npm run eslint -- --quiet",
     "eslint-fix": "npm run eslint -- --fix",
     "local-ci": "npm run eslint && npm run test-functional && npm audit --audit-level=low && npm run validate-links",
-    "build": "dotenv -e .env -- cross-env NODE_ENV=production webpack --config ./webpack.config.js",
-    "build-dev": "dotenv -e .env -- cross-env NODE_ENV=development webpack --config ./webpack.config.js",
-    "dev-utils": "dotenv -e .env -- cross-env NODE_ENV=development NAME=utils ENTRY=./src/browser/utils/index.js nodemon --trace-warnings --verbose --watch ./src/shared --watch ./src/browser/utils --delay 2500ms ./bin/dev.js -e js,css",
-    "dev-game": "dotenv -e .env -- cross-env NODE_ENV=development NAME=game ENTRY=./src/browser/game/index.js nodemon --trace-warnings --verbose --watch ./src --delay 2500ms ./bin/dev.js -e js,css",
-    "dev-editor": "dotenv -e .env -- cross-env NODE_ENV=development NAME=editor ENTRY=./src/browser/editor/index.js nodemon --trace-warnings --verbose --watch ./src --delay 2500ms ./bin/dev.js -e js,css",
-    "dev-backend": "dotenv -e .env -- cross-env PORT=8000 NODE_ENV=development nodemon --trace-warnings --verbose --watch ./bin/backend --watch ./src/shared --delay 2500ms ./bin/backend/index.js -e js",
+    "dev-build-utils": "dotenv -e .env -- cross-env NODE_ENV=development NAME=utils ENTRY=./src/browser/utils/index.js nodemon --trace-warnings --verbose --watch ./src/shared --watch ./src/browser/utils --delay 2500ms ./bin/buildDev.js -e js,css",
```

Comment:
why have you changed these scripts ? actually dev-game and so on was the correct terminoly since this is not only building a bundle but also running a watched routine

Reply:
I change the name because it's confusing with the script `dev-backend` which are not realted to the script `dev.js`.

Reply:
Ok that's make sense, I guess the common part is the watch routine. Do you think dev should be replace by watch ? Both solution are fine for me